core/src/main/java/org/mule/exception/RollbackMessagingExceptionStrategy.java

```diff
@@ -18,7 +21,14 @@
 {
     private RedeliveryExceeded redeliveryExceeded;
     private Integer maxRedeliveryAttempts;
-    
+
+
+    @Override
+    protected void doInitialise(MuleContext muleContext) throws InitialisationException
+    {
+        LifecycleUtils.initialiseIfNeeded(redeliveryExceeded);
```

Comment:
Yes, I will work in blog posts and documentation for this, but hopefully users are not so impacted by this because users normally don't develop components that are to be so in charge of their own lifecycle.